src/mento.ts

```diff
@@ -4,7 +4,7 @@ import {
   IExchangeProvider,
   IExchangeProvider__factory,
 } from '@mento-protocol/mento-core-ts'
-import { BigNumber, providers, Signer } from 'ethers'
+import { BigNumber, BigNumberish, providers, Signer } from 'ethers'
```

Comment:
do you still need to import BigNumber here?

Reply:
Yes because the getAmountIn/out methods still return a BigNumber. If it were not used, VSCode would have auto removed it :)